internal/exec/stages/disks/luks.go

```diff
@@ -282,6 +285,21 @@ func (s *stage) createLuks(config types.Config) error {
 				config = string(clevisJson)
 			}
 
+			if len(luks.Clevis.Tang) > 0 {
+				// We cannot guarantee that networking is up yet, fetch the tang
+				// server advertisement to ensure that networking is up before
+				// we pass the device to clevis
+				u, err := url.Parse(luks.Clevis.Tang[0].URL)
```

Comment:
Strictly speaking, this isn't enough.  `Tang[1]` might be on a different NIC that hasn't come up yet.  How would you feel about doing one fetch for each Tang server in the list?